src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Seamlessly adopt New Relic Service catalog with the GitHub integration
+tags:
+  - New Relic integrations
+  - GitHub integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Implementing a New Relic Service Catalog can unlock considerable benefits, but at present, it requires a considerable investment of time and resources. You have to take care of team orchestration, user provisioning, role assignments, and entity ownership delineation, which create a significant threshold for entry. To ease this challenge, we suggest the adoption of an automation solution that capitalizes on existing team configurations and relationships maintained in external systems such as GitHub. This simplifies the initial configuration, dramatically lessen the required effort, and accelerates the adoption of the New Relic Service Catalog, making its powerful insights and organization tools more accessible to users.
```

Comment:
Implementing a New Relic Service Catalog can unlock considerable benefits, but it requires considerable time and resources. You have to take care of team orchestration, user provisioning, role assignments, and entity ownership delineation. These create a significant threshold for entry. To ease this challenge, consider an automation solution that capitalizes on existing team configurations and relationships maintained in external systems such as GitHub. This simplifies the initial configuration and accelerates the adoption of the New Relic Service Catalog.